.github/workflows/docker-build-pr.yml

```diff
@@ -1,4 +1,4 @@
-name: Docker Build and Push
+name: Docker Build and PR
```

Comment:
Change it to just `Docker Build` or `Docker Build PR`. `Build and PR` doesn't make sense.

Reply:
build and pull request?